handbook/engineering/search/index.md

```diff
@@ -2,9 +2,99 @@
 
 We own the end-to-end Sourcegraph search experience.
 
+## Vision
+
+Sourcegraph search is:
+
+- **Fast:** Performance of showing results is fast and I can get to what I’m looking for quickly.
+- **Expressive:** It is possible to construct a search query to find exactly the results I’m looking for. The tools, syntax, filters, are provided to construct the queries I want to write. We strive for a search experience where the answer to “Can I do ‘X’ with Sourcegraph?” will always be yes.
+- **Easy to use:** Users can quickly understand how to find what they are looking for and what options are available for searching. The search syntax is clear and intuitive.
+- **Shareable:** Sourcegraph searches are easy to share, and provide team and organization wide value. This in turn creates network effects that compound the value Sourcegraph provides with scale.
+
 ## Goals
 
-Our current goals are documented in the [search team's current tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+_Updated 2020-08-27_
+
+## Long term (~1 year)
+
+Sourcegraph search is far and away the best code search solution, search results are fast, and I can find what I’m looking for quickly.
```

Comment:
I am wondering whether this long-term goal adds value when it comes to guiding the team's work. It's part outcome of achieving all four pillars of the vision (fast+expressive+easy to use+shareable search = Sourcegraph is far and away the best code search solution), part unquantified focus on certain pillars (search results are fast -> fast, I can find what I'm looking for quickly -> expressive + easy to use).

Is there a more concrete outcome we could/should aim for here?

Reply:
I agree, I think the vision is clear and this long term section doesn't add anything. I would delete this section.